autoware_perception_msgs/msg/ObjectClassification.msg

```diff
@@ -6,6 +6,10 @@ uint8 TRAILER = 4
 uint8 MOTORCYCLE = 5
 uint8 BICYCLE = 6
 uint8 PEDESTRIAN = 7
+uint8 ANIMAL = 8
+uint8 HAZARD = 9
+uint8 OVER_DRIVABLE = 10
+uint8 UNDER_DRIVABLE = 11
```

Comment:
I believe these labels are made up words it might be better if we put some comments explaining what these classes are for with some examples

Reply:
These variables appear in sensor datasheets without further information (this was discussed previously in this PR).
That being said, they are not being used in the data that we have. To keep things compatible and "universal" I decided to include them for now.

My interpretation based purely on intuition:
 - over_drivable: objects that exist, but  can be driven over. Example: leaves
 - under_dirvable: objects that exist, but the vehicle can pass under. Example: under passes

Reply:
Could you put those information as a comment then?

Reply:
Added them in https://github.com/autowarefoundation/autoware_msgs/pull/120/commits/3dffd270bc57e8ff0541ae47b4dc9fb80c41d23d